test/e2e/installplan_e2e_test.go

```diff
@@ -3716,24 +3691,22 @@ var _ = Describe("Install Plan", func() {
 		Expect(err).ToNot(HaveOccurred())
 		defer c.KubernetesInterface().RbacV1().ClusterRoles().Delete(context.Background(), role.GetName(), metav1.DeleteOptions{})
 
-		// create operator group referencing the SA
-		og := &operatorsv1.OperatorGroup{
-			ObjectMeta: metav1.ObjectMeta{
-				Name:      genName("og-"),
-				Namespace: generatedNamespace.GetName(),
-			},
-			Spec: operatorsv1.OperatorGroupSpec{
-				ServiceAccountName: sa.GetName(),
-			},
-		}
-		_, err = crc.OperatorsV1().OperatorGroups(generatedNamespace.GetName()).Create(context.Background(), og, metav1.CreateOptions{})
-		Expect(err).ToNot(HaveOccurred())
+		// Update the existing OG to use the ServiceAccount
```

Comment:
(just reviewing the last commit) really not clear where the existing OG comes from - why is it shared? Can it be test-local?

Reply:
[This line](https://github.com/operator-framework/operator-lifecycle-manager/pull/3042/files#diff-eddfb8217b77c532b776defca62eb7d6e4710bb15ceb8bf8a4d8de408df3a6a4R72) is ran before each it, which creates the namespace and operatorGroup, so we don't need to create new resources here.